plugins/python_3_script/help.md

```diff
@@ -134,6 +134,7 @@ If installation fails, try increasing the `Timeout` connection input to `900` (1
 
 # Version History
 
+* 4.0.9 - Updated the SDK to latest version
```

Comment:
one minor, should we say 'to fix memory leak issue'? will just help when looking back over the change log what the purpose of getting the latest SDK is. but not a blocking comment!